src/assets/courses/rosters.json

```diff
@@ -1 +1 @@
-["FA14","WI15","SP15","SU15","FA15","WI16","SP16","SU16","FA16","WI17","SP17","SU17","FA17","WI18","SP18","SU18","FA18","WI19","SP19","SU19","FA19","WI20","SP20","SU20","FA20","WI21","SP21","SU21"]
\ No newline at end of file
+["WI17","SP17","SU17","FA17","WI18","SP18","SU18","FA18","WI19","SP19","SU19","FA19","WI20","SP20","SU20","FA20","WI21","SP21","SU21"]
```

Comment:
Can we add a newline at the end of this file?

Reply:
@tcho6319 new line added